tests/fixtures/TagLib.Tests.FileFormats/AsfFormatTest.cs

```diff
@@ -41,8 +41,14 @@ public void WriteStandardTags ()
         {
             StandardTests.WriteStandardTags (sample_file, tmp_file);
         }
-        
-        [Test]
+
+				[Test]
+				public void WriteExtendedTags()
+				{
+					ExtendedTests.WriteExtendedTags(sample_file, tmp_file);
+				}
+
+				[Test]
```

Comment:
I think this project is largely using spaces and not tabs. This causes the indenting to be all weird.